lib/slack-rtmapi.rb

```diff
@@ -17,4 +23,39 @@ def self.get_url(options)
     URI(body['url'])
   end
 
+  def initialize(options)
+    @token = options.fetch(:token)
+    @api_url = options[:slack_api_url] || 'https://slack.com/api'
+  end
+
+  def start_rtm
+    @data = SlackWebAPI.call(method: 'rtm.start', token: @token)
+    @websocket_url = URI(@data['url'])
+    client
+  end
+
+  def client
+    @client ||= SlackRTM::Client.new websocket_url: @websocket_url
+  end
+
+  def channel_id(channel_name)
+    channels = @data['channels']
+    channel_data = channels.find { |channel| channel['name'] == channel_name }
+    channel_data['id'] unless channel_data.nil?
+  end
+end
+
+#
+# SlackWebAPI
+#
+module SlackWebAPI
```

Comment:
Some remarks:
- modules should be in their own files. Please move that module to `lib/<camelCaseNameOfModule>.rb`
- as the other module is named `SlackRTM`, I prefer to name this one `SlackWeb`, which would be used `SlackWeb::Client` (also, it resolve the conflict with the other gem named "slack-webapi")